### PR TITLE
Doc/add examples

### PR DESCRIPTION
**Pull request by shtrom**
Some examples to match the PHP and Ruby SDKs.

Not sure whether we actually want examples in the SDK source, though.

### CODE REVIEW

**Comment by amlearn** [comment]
we got rid of the makefile?

**Comment by amlearn** [comment]
#derp... actually yeah, it's just the single commit

File: examples/simple/init_data.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+i = 0
+data_request = { "limit": 2 }
+for result in client.request_iter(itembankUri, security, consumer_secret, data_request, action):
+    # print the length of each page of the items list (will print a line for each page in the results)
+    print(">>> [%s (%d)] %s" % (
+        itembankUri,
+        i,
+        json.dumps(item)
+        ))
+    if i > 5:
```
[the remark this answers is not in the document]
Comment (alangarf) [reply examples/simple/init_data.py]: What is that `i` even doing? And same above, it's iterating on a iterable thing, why do we care about `i`?

**Comment by shtrom** [comment]
Fixed, rebased, and bundled a couple of additional cosmetic fixups. You can test the examples with

```make
. .venv/Darwin-x86_64-sdk-python/bin/activate
./examples/simple/init_items.py
[...]
./examples/simple/init_data.py
[...]
```

File: examples/simple/init_data.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (alangarf) [reply examples/simple/init_data.py]: Oh, it's in the print output.

```
for i, result in enumerate(client.request_iter(itembankUri, security, consumer_secret, data_request, action)):
```

[the remark this answers is not in the document]
Comment (shtrom) [reply examples/simple/init_data.py]: 🤦‍♂️

[the remark this answers is not in the document]
Comment (shtrom) [reply examples/simple/init_data.py]: I is just an arbitrary limit so we don't get output forever.

**Comment by stellalie** [comment]
@shtrom Thanks for that, we are bit swamped at the moment, and will get someone from Turtle take a look at this later at the end of release. Cheers!

**Pull request merged**